## Brooke Weinstein, Zongji Wang, Qi Zhou, Scott William Roy. (2025) ZW Sex Chromosome Differentiation in Palaeognathous Birds Is Associated with Mitochondrial Effective Population Size but Not Mitochondrial Genome Size or Mutation Rate. Genome Biology & Evolution. DOI:[10.1093/gbe/evaf005]

This repository contains the data and supplemental information from our study, which investigated the relationship between the expansion of the non-recombining W chromosome and mitochondrial genome evolution in palaeognathous birds.

### Abstract

Eukaryotic genome size varies considerably, even among closely related species. The causes of this variation are unclear, but weak selection against supposedly costly “extra” genomic sequences has been central to the debate for over 50 years. The mutational hazard hypothesis, which focuses on the increased mutation rate to null alleles in superfluous sequences, is particularly influential, though challenging to test. This study examines the sex chromosomes and mitochondrial genomes of 15 flightless or semiflighted palaeognathous bird species. In this clade, the nonrecombining portion of the W chromosome has independently expanded stepwise in multiple lineages. Given the shared maternal inheritance of the W chromosome and mitochondria, theory predicts that mitochondrial effective population size ($N_e$) should decrease due to increased Hill–Robertson interference in lineages with expanded nonrecombining W regions. Our findings support the extent of the nonrecombining W region with three indicators of reduced selective efficiency: (i) the ratio of nonsynonymous to synonymous nucleotide changes in the mitochondrion, (ii) the probability of radical amino acid changes, and (iii) the number of ancient, W-linked genes lost through evolution. Next, we tested whether reduced $N_e$ affects mitochondrial genome size, as predicted by weak selection against genome expansion. We find no support for a relationship between mitochondrial genome size and expanded nonrecombining W regions, nor with increased mitochondrial mutation rates (predicted to modulate selective costs). These results highlight the utility of nonrecombining regions and mitochondrial genomes for studying genome evolution and challenge the general idea of a negative relation between the efficacy of selection and genome size.

The primary datasets include:

* Mitochondrial genome sequences (`mt.genome`)
* Phylogenetic tree (`bird.tre`)
* Data on radical amino acid changes (`is_change_rad.csv`)
* Raw data on palaeognath species (`paleognath_raw_data.csv`)

The analyses were conducted using tools like HyPhy for ancestral state reconstruction and other R scripts.

## Contact

For any questions or inquiries, please contact:

Brooke Weinstein
bweinstein2@ucmerced.edu
